Documentation/guides/Deploying Your Application.md

```diff
@@ -199,12 +245,403 @@ Figure 7 – Standalone Settings – Linux
+* call JavaScript from LiveCode Builder widgets and extensions using the
+com.livecode.emscripten module
+* implement widgets by embedding DOM elements as native layers
+* perform basic networking operations using the **load** command
+
+Several important features are not yet supported:
+
+* multimedia (the "player" control)
+
+Two important unsupported features are unlikely to be added in the near future:
+
+* operations that need to pause the script while something happens
+  (e.g. `wait 10`)
+* externals (including revdb)
+
+##### Contents of the HTML5 standalone
+
+The HTML5 standalone contains four files:
+
+* A standalone archive, named `standalone.zip` by default.  This file contains
+your application and all of the resources that it depends on.  When the engine
+runs, the filesystem that's visible to the engine (e.g. via the `open file`
+syntax) is based on the contents of the standalone archive.
+
+* The engine itself, which consists of two files.  The `.js` file contains the
+engine's executable code, and the `.html.mem` file contains essential data
+that's needed for the engine to run.  These files are always the same, and only
+change when LiveCode is upgraded.
+
+* A test HTML page.  This can be opened in a browser and will correctly prepare,
+download and start your HTML5 app in a convenient test environment.
+
+##### Advanced: HTML5 standalone filesystem
+
+JavaScript applications running in a browser don't have access to the host
+system's filesystem.  Instead, the filesystem-related features of LiveCode, such
+as `open file`, use a virtual filesystem (VFS) that exists only in memory.  This
+filesystem is initialised before the engine starts, and is reset and its content
+discarded when the engine stops (when the user closes the browser view or
+navigates to a different page).
+
+During engine startup, the VFS is populated from the contents of the
+`standalone.zip` file that's created by the HTML5 deployment process.  All of
+the initial files are stored in `/boot/` in the VFS.
+
+There are several special files & directories in the `/boot/` directory:
+
+* `/boot/__startup.livecode`: a stack that performs loading & initialisation
+operations during engine startup.
+* `/boot/auxiliary_stackfiles/`: during startup, each file in this directory
+is loaded and is receives the `revLoadLibrary` message
+* `/boot/fonts/basefont.ttf`: the font used by the engine
+* `/boot/standalone/__boot.livecode`: this is the initial stack of the
+application, which is the main stack that receives the `startup` message when
+the engine has been initialized
+* `/boot/standalone/`: the `defaultFolder` when the engine starts, and the
+location where additional assets selected using the "Copy files" page of the
+standalone builder are placed
+* `/boot/extensions/extensions.txt`: list of extensions included in the
+standalone, in the order in which they should be autoloaded
+* `/boot/extensions/`: the directory where all autoloaded extensions are stored
```

Comment:
I don't think this info is correct any more since we switched to using the capsule stuff for emscripten deployment

Reply:
Oh... OK I will check the standalone builder. I do vaguely recall that change.